### ISSUE TITLE
train with deeper layers for imagenet

### ISSUE THREAD

**Issue by zjupx** [wy1iu/DCNets#2]
thank you for your greate jobs.  I see you report the result with res-18 for imagenet in your paper. Do you have trained more deep layers for imagenet or some other largescale dataset? for example res101, or res152 ? can you share some experience for that?

**Comment by wy1iu** [comment]
Yes, you can train deeper networks. But you may need to modify the code to support multi-gpu setting for more gpu memory. We have tried ResNet-34 and ResNet-50, and they can work well. Due to resource limits, we have not tried ResNet-101 and ResNet-152, but I think it will work well too.
